ethapi/backend.go

```diff
@@ -99,7 +99,7 @@ type Backend interface {
 	ServiceFilter(ctx context.Context, session *bloombits.MatcherSession)
 }
 
-func GetAPIs(apiBackend Backend) []rpc.API {
+func GetAPIs(apiBackend Backend, chain *core.BlockChain) []rpc.API {
```

Comment:
hard type, would make unusable for Polaris because we dont use/have the `*core.BlockChain` object